docs/administration/installation/settings_main_topics.md

```diff
@@ -627,7 +627,8 @@ email.reply.imap.use_ssl = true
 email.reply.imap.use_idle = true
 ```
 
-Don't forget to start mail_fetcher daemon, documentation here [Backend README](/docs/development/README.md) and chapter "Run daemons according to your config"
+Don't forget to start mail_fetcher daemon, documentation here [Backend advanced setup](/docs/development/advanced/backend_server_setup.md)
+and chapter "Running the Tracim Backend Daemons"
```

Comment:
why text on 2 lines here ?
Missing `.` at end.

Reply:
The line break is for readability, to avoid too long lines is source code.

When viewing the md file converted in html, the line break won't exist.

Reply:
will fix the dot

Reply:
fixed